diff --git a/dots/.config/hypr/hyprland/colors.lua b/dots/.config/hypr/hyprland/colors.lua
index b5ad1ce1..fec6a0cb 100644
--- a/dots/.config/hypr/hyprland/colors.lua
+++ b/dots/.config/hypr/hyprland/colors.lua
@@ -1,16 +1,16 @@
 hl.config({
     general = {
         col = {
-            active_border   = "rgba(44464f77)",
-            inactive_border = "rgba(1a1b2033)",
+            active_border   = "rgba(3f475977)",
+            inactive_border = "rgba(181b2533)",
         },
     },
     misc = {
-        background_color = "rgba(121318FF)",
+        background_color = "rgba(10131cFF)",
     },
 })
 
-hl.window_rule({ -- not sure how to syntax "pin 1"
+hl.window_rule({
     match        = { pin = 1 },
-    border_color = "rgba(afc6ffAA) rgba(afc6ff77)",
+    border_color = "rgba(56d6f5AA) rgba(56d6f577)",
 })
